units/XNB2301/XNB2301_script.lua

```diff
@@ -1,14 +1,13 @@
--- T2 PD
-
 local NStructureUnit = import('/lua/nomadsunits.lua').NStructureUnit
 local KineticCannon1 = import('/lua/nomadsweapons.lua').KineticCannon1
 
+--- Tech 2 Point Defence
```

Comment:
The game uses American English spelling
```suggestion
--- Tech 2 Point Defense
```